components/prices/PriceCard.js

```diff
@@ -216,28 +216,42 @@ export default function PriceCard({
             </div>
           </div>
         </div>
-
+      </div>
+      <div
+        tabIndex={0}
+        className="flex w-full flex-row justify-center bg-white px-4 pb-4"
+        // title={price <= 0 ? contentText.cta.signUpNow : contentText.cta.get + ' ' + storage}
+      >
         <div
-          tabIndex={0}
           onClick={() => {
-            checkout({
-              planId: getPlanId(stripeObject),
-              mode: billingFrequency === -1 ? 'payment' : 'subscription',
-            });
+            if (cta[1].startsWith('https')) {
```

Comment:
why?

Reply:
Because when the user clicks on free plan, the button does nothing.